tools/gherkinize.py

```diff
@@ -0,0 +1,911 @@
+import argparse
+from datetime import datetime, timedelta, timezone
+from io import open
+import logging
+from os import path
+from pathlib import Path
+import re
+import sys
+from typing import Any, Dict, List, Literal, Optional, Union, overload
+from typing_extensions import Self
+from jinja2 import Environment, FileSystemLoader
+from cel.expr import checked_pb2, eval_pb2, value_pb2
+from cel.expr.conformance.test import simple_pb2
+from cel.expr.conformance.proto2 import (
+    test_all_types_pb2 as proto2_test_all_types,  # noqa
+    test_all_types_extensions_pb2 as proto2_test_all_types_extensions,  # noqa
+)
+from cel.expr.conformance.proto3 import test_all_types_pb2 as proto3_test_all_types  # noqa
+from google.protobuf import (
+    any_pb2,
+    descriptor_pool,
+    descriptor,  # noqa
+    duration_pb2,
+    message_factory,
+    message,
+    struct_pb2,
+    symbol_database,  # noqa
```

Comment:
are these `noqa` imports actually used? side effects?

Reply:
ah, yeah that's reasonable. I think the only thing you could do is get more specific w/ the code and maybe an explanation afterwards in the comment (in protovalidate we add the code [here](https://github.com/bufbuild/protovalidate-python/blob/1f0d1837077765e4c21933202594bddb9f065b5a/test/conformance/runner.py#L23-L51)).